### ISSUE TITLE
Feature request: More D&D blog posts

### ISSUE THREAD

**Issue by protherj** [pbausch/onfocus#1]

**Comment by pbausch** [comment]
can't reproduce, closing.

**Comment by pbausch** [comment]
You're right though, I've been all covid-19 since March. I need to get more posts happening in the [D&D tag](https://www.onfocus.com/tag/dnd). Not to mention getting our game back on track. 😱

**Comment by protherj** [comment]
Heh, and here I thought I was just joking around :)

**Comment by pbausch** [comment]
I know you were! I'm joking too and trying out all the bells and whistles in the Issues section. I don't get many issues around here!
